src/parser/classes/livechat/items/LiveChatSponsorshipsHeader.ts

```diff
@@ -1,29 +1,20 @@
-import { Parser } from '../../../index.js';
 import { YTNode } from '../../../helpers.js';
-import type { ObservedArray } from '../../../helpers.js';
 import type { RawNode } from '../../../index.js';
 import NavigationEndpoint from '../../NavigationEndpoint.js';
 import Text from '../../misc/Text.js';
 import Thumbnail from '../../misc/Thumbnail.js';
-import LiveChatAuthorBadge from '../../LiveChatAuthorBadge.js';
 
 export default class LiveChatSponsorshipsHeader extends YTNode {
   static type = 'LiveChatSponsorshipsHeader';
 
-  author_name: Text;
-  author_photo: Thumbnail[];
   primary_text: Text;
-  author_badges: ObservedArray<LiveChatAuthorBadge> | null;
   menu_endpoint: NavigationEndpoint;
   context_menu_accessibility_label: string;
   image: Thumbnail[];
 
   constructor(data: RawNode) {
     super();
-    this.author_name = new Text(data.authorName);
```

Comment:
I think it's fine to keep these fields here. They're part of the node after all, and people might want to access it (like using YouTube.js' parser externally, or other nodes might use this header).

Reply:
Thanks for the feedback! I've restored these fields as suggested!

Reply:
I'm really sorry, I didn't realise that the node didn't have access to the channel ID when I made the suggestion. Maybe it's better to remove the Author stuff again as it relies mostly on fields in a different node.

Reply:
No worries at all! I really appreciate you review my code and give me feedbacks!